odb-compiler/CMakeLists.txt

```diff
@@ -177,9 +177,16 @@ set_target_properties (odb-compiler
         RUNTIME_OUTPUT_DIRECTORY ${ODB_RUNTIME_DIR})
 target_include_directories (odb-compiler PUBLIC ${LLVM_INCLUDE_DIRS})
 target_compile_definitions (odb-compiler PUBLIC ${LLVM_DEFINITIONS})
-llvm_map_components_to_libnames (llvm_libs core bitwriter x86codegen)
-target_link_directories (odb-compiler PUBLIC ${LLVM_LIBRARY_DIRS})
-target_link_libraries (odb-compiler PUBLIC ${llvm_libs})
+
+# This is apparently deprecated since 4 years
+# https://github.com/llvm-mirror/llvm/blob/master/cmake/modules/LLVM-Config.cmake#L67
+#
+#llvm_map_components_to_libnames (llvm_libs core bitwriter x86codegen)
+#target_link_directories (odb-compiler PUBLIC ${LLVM_LIBRARY_DIRS})
+#target_link_libraries (odb-compiler PUBLIC ${llvm_libs})
+
+# This is how you're supposed to do it by reading LLVM-Config.cmake
```

Comment:
probably worth just deleting these comments to avoid confusion. we're never going to change this anyway.